### PR TITLE
[sample] New sample code of building a TH02 based thermostat

### PR DESCRIPTION
**Pull request by ttzeng**
This adds example code of building a ZJS based thermostat using the Grove Temperature & Humidity Sensor.

Signed-off-by: Tonny Tzeng tonny.tzeng@intel.com

### CODE REVIEW

**Comment by ttzeng** [comment]
@gvancuts @grgustaf @poussa PTAL, these are sample code we used in the tutorial. Thanks in advance.

File: samples/thermostat/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Example code of building a thermostat using ZJS
+===
+
+This folder contains the example code for building a ZJS based thermostat using the [Grove Temperature & Humidity Sensor]*. The example code `th02.js` serves as a template that reads the sensed temperature and humidity data from the sensor, then the example code `th02-ess.js` and `th02-ocf.js` expose the sensor data through Bluetooth [Environmental Sensing Service] (ESS) and [OCF] networking protocol respectively using the [ZJS communications API].
+
+| Filename | Description |
+|----------|-------------|
+| [th02.js](./th02.js)     | Read the environmental data from the sensor and display the data on the serial console |
```
Comment (grgustaf) [inline samples/thermostat/README.md]: Remove extra space at the end of the line, this made it fail our checks.

**Comment by grgustaf** [comment]
I remembered I can just fix it myself. :)

**Comment by grgustaf** [comment]
+1

**Comment by gvancuts** [comment]
Thanks @grgustaf !

**Pull request merged**